docs/content/tools/_index.md

```diff
@@ -51,24 +44,12 @@ Details on acquiring and running the converters:
 - converting *into* JSON - [XML to JSON](https://github.com/usnistgov/OSCAL/tree/main/json)
 - converting *into* XML - [JSON to XML](https://github.com/usnistgov/OSCAL/tree/main/xml)
  
-## OSCAL Application Frameworks, Tools and Libraries
+## NIST's OSCAL Application Frameworks, Tools and Libraries
 
 Valid OSCAL is open-ended in application. Some of the tools described provide validation and conversion, while others do not, presuming that inputs are already valid.
```

Comment:
So we are not including `oscal-cli` yet? If so, we should add it.

Reply:
Added